app/assets/javascripts/osom-tables.js

```diff
@@ -20,6 +20,20 @@
     order_table(this);
   });
 
+  $(document).on('click', '.osom-table table th.mark > input[type="checkbox"]', function() {
```

Comment:
Can you maybe move the callback function here into a named variable, so it's easy to understand what it does? E.g.

```
$(document).on('click', '.osom-table table th.mark > input[type="checkbox"]', someFunc)
```

I can't quite wrap my head around it.